doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,87 @@
+- **Spec Status:** Waiting for approve
+
+## What is GitHub Discussions
+
+GitHub Discussions is a feature that currently is in closed beta status. It will be released until the end of the year. 
+
+It's a new tab in the GitHub repo page where people can start new threads like in a forum.
+
+The threads must be added to a category. The answer feature can be enabled or disabled for categorys. If enabled a person with triage permission or the creator of the thread can mark a reply as answer.
+
+You can sort threads in the overview for categories and for (un-)answered.
+
+One thread can be pinned on the main page of the discussion section. (One with rules or something like that).
+
+Threads uses the same numeration like PR and issues. So they can be linked by the others. A conversation can also be locked like in issues and PR.
+
+## Why do is this for PowerToys important
+
+1. Sometimes people just want to say thank you to the PT team (like in https://github.com/microsoft/PowerToys/issues/7873) and a thread in GitHub Discussions where every one can comment if he just want to say thank you, is much better for the users and the team than open every time a new issue for that.
+
+2. Questions: If people just have questions, it gets difficult by asking them in issues. It's more useful for both sides (team and user) when questions are asked in a different section. Because in GitHub discussion you can mark a reply as the answer and other users can find the answer quicker.
+
+3. Off-topic. Sometimes issue discussions get very and sometimes a little off-topic (like https://github.com/microsoft/PowerToys/issues/7916#issuecomment-724813783) And it would be better if you can open a thread and write "As seen in #XXXX I think the real reason for ... is ..."
+
+4. Tell informations to the community: 
+
+4. More community interaction: More information in the underneath section
+
+### Community interaction
+
+#### Do PowerToys needs this?
+
+Yes because PowerToys is a community project. And PowerToys is a project for power users. And not all power users are programers (like designers, etc.). And in the moment the community interaction is mostly based on programing.
+
+#### But PowerToys has already community interaction. PowerToys has around 200 new issues in a week.
+
+That's right but these are normally bug reports and feature suggestions and PowerToys needs more things for a strong community. Things like jokes, memes and grand discussion about PowerToys related things.
+
+## Does this hinder the PowerToys team from working on PowerToys?
+
+Not really. One of the core team members needs to set the settings first. But after that the triage team can take care of this section. Most of them don't program for PowerToys so they don't lose time for programing. The programers of PowerToys don't need to take care of GitHub discussion, because it's only about community stuff where the triage team don't need the help of them.
```

Comment:
"programming". 

PS: Well I do program but not with the technologies used by PowerToys (I am learning C# and .NET though). But I do keep an eye on the PowerToys repo while I code.

Reply:
"programmers"